crates/ironrdp-async/src/connector.rs

```diff
@@ -53,13 +53,17 @@ where
 {
     let mut buf = WriteBuf::new();
 
-    debug!("CredSSP procedure");
+    if connector.is_credssp_step() {
+        debug!("CredSSP procedure");
+    }
 
     while connector.is_credssp_step() {
         single_connect_step(framed, &mut connector, &mut buf).await?;
     }
 
-    debug!("Remaining of connection sequence");
+    if connector.is_credssp_step() {
+        debug!("CredSSP complete, continue with remainder of connection sequence");
+    }
```

Comment:
This is surprising. Is it a leftover?

Reply:
Ah yeah, derp. That's what I meant.

Reply:
How's this: 2595917e64712d4c4f4b497b903ce998d196fc1e

Reply:
Good for me!